reana_server/rest/users.py

```diff
@@ -333,6 +343,7 @@ def get_you():
                         "full_name": me.full_name,
                         "username": me.username,
                         "disk_usage": me.get_user_disk_usage(),
+                        "cpu_usage": me.get_user_cpu_usage(),
```

Comment:
A bit off topic question - I wonder when we should combine quotas into some kind of dict (maybe something similar like [here](https://github.com/reanahub/reana-client/pull/425/files#diff-4f27f08c3f46f0b755c1d5ac1a1e2dcaR72)) and group the usage and limits of different quota resources. Maybe we should create an issue for that?

Reply:
I think this is a great idea! Maybe we can do a little refactor once we merge these PRs so we do not block the rest? Can you create a new issue for this from your comment?